src/lib/actions/RulesAction.ts

```diff
@@ -41,6 +41,8 @@ export class RulesAction {
 
 	public async execute(input: RulesInput): Promise<void> {
 		const config: CodeAnalyzerConfig = this.dependencies.configFactory.create(input['config-file']);
+		//console.log(`selectors: ${JSON.stringify(input["rule-selector"])}`);
```

Comment:
These consoles can be removed - is the rule selector no longer valid here, or does the logic need to be re-implemented?

Reply:
Good catch. I was trying to debug the problem I found on Friday and had these left in there. I'll take them out.